src/views/settings/Feature/Chat.vue

```diff
@@ -10,113 +10,62 @@
+        'CHAT_AI_EMBED_URL'
       ],
       fieldsMeta: {
         CHAT_AI_TYPE: {
           hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api'
-          }
-        },
-        GPT_BASE_URL: {
-          el: {
-            autocomplete: 'new-password'
-          },
-          hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api' || formValue.CHAT_AI_TYPE !== 'gpt'
-          }
-        },
-        GPT_API_KEY: {
-          el: {
-            autocomplete: 'new-password'
-          },
-          hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api' || formValue.CHAT_AI_TYPE !== 'gpt'
-          }
-        },
-        GPT_PROXY: {
-          hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api' || formValue.CHAT_AI_TYPE !== 'gpt'
+            return formValue.CHAT_AI_METHOD !== 'api' || hasProviders(formValue)
           }
         },
-        GPT_MODEL: {
-          hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api' || formValue.CHAT_AI_TYPE !== 'gpt'
-          }
-        },
-        DEEPSEEK_BASE_URL: {
-          el: {
-            autocomplete: 'new-password'
-          },
-          hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api' || formValue.CHAT_AI_TYPE !== 'deep-seek'
-          }
-        },
-        DEEPSEEK_API_KEY: {
-          el: {
-            autocomplete: 'new-password'
-          },
-          hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api' || formValue.CHAT_AI_TYPE !== 'deep-seek'
-          }
-        },
-        DEEPSEEK_PROXY: {
-          hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api' || formValue.CHAT_AI_TYPE !== 'deep-seek'
-          }
-        },
-        DEEPSEEK_MODEL: {
-          hidden: (formValue) => {
-            return formValue.CHAT_AI_METHOD !== 'api' || formValue.CHAT_AI_TYPE !== 'deep-seek'
-          }
+        CHAT_AI_PROVIDERS: {
+          component: ChatProvidersField,
+          hidden: (formValue) => formValue.CHAT_AI_METHOD !== 'api'
         },
         CHAT_AI_EMBED_URL: {
           hidden: (formValue) => formValue.CHAT_AI_METHOD !== 'embed'
         }
       },
+      afterGetFormValue(formValue) {
+        const providers = Array.isArray(formValue.CHAT_AI_PROVIDERS) ? formValue.CHAT_AI_PROVIDERS : []
+        return {
+          ...formValue,
+          CHAT_AI_PROVIDERS: {
+            providers: providers
+          }
+        }
+      },
+      cleanFormValue(values) {
+        const config = values.CHAT_AI_PROVIDERS || {}
+        const providers = Array.isArray(config.providers) ? config.providers : []
+        return {
+          ...values,
+          CHAT_AI_PROVIDERS: providers,
+          CHAT_AI_TYPE: values.CHAT_AI_TYPE || providers[0]?.type || values.CHAT_AI_TYPE
+        }
+      },
       submitMethod() {
         return 'patch'
       }
```

Comment:
This is a Vuex application component related to chat settings. Please share the context of your concern so that I can offer more specific assistance.